glue/egmde.remote

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
+if [ $# -lt 1 -o "$1" == "--help" -o "$1" == "-h" ]
+then
+  echo "$(basename $0) - Handy launch script for using egmde over VNC"
+  echo "  This sets up ssh tunnelling and a remote VNC server, once the"
+  echo "  server is running it can be connected to locally with any VNC"
+  echo "  client (e.g. 'gvncviewer localhost')"
+  echo "Usage: $(basename $0) [ssh-login]"
+  echo "[ssh-login] login for ssh: hostname, user@hostname or -p port user@ipaddress"
+  echo "The desktop size can be set with EGMDE_REMOTE_OUTPUT=WxH $(basename $0) [login]"
+  exit
+fi
+
+ssh -T -L 5900:localhost:5900 $* -- bash -el <<EOT
```

Comment:
Shouldn't we pass the args as is?

```suggestion
ssh -T -L 5900:localhost:5900 "$@" -- bash -el <<EOT
```

Reply:
I think that failed with `-p 10022 alan-griffiths@localhost`. (Will check later)

Reply:
No, you are right! (I know, you knew that.)